### ISSUE TITLE
bun - ENOENT: Failed to connect to Soundpad pipe using soundpad.js on Windows

### ISSUE THREAD

**Issue by foorb** [Seblor/soundpad.js#1]
Hi, 
I'm encountering an issue when trying to use the soundpad.js package with bun to connect to the Soundpad pipe on Windows. **While manually connecting to the pipe or using nodejs works**, soundpad.js fail here with bun.
I use the basic example for testing. With` await openSoundpad()` its just waiting, but soundpad is started. When removing `await openSoundpad()` i am getting the following error.

```
PS C:\Users\foorb\workspace\tsba> bun .\test.ts
C:\Program Files\Soundpad\Soundpad.exe
54 |             if (this.options.startSoundpadOnConnect) {
55 |                 yield this.openSoundpad();
56 |             }
57 |             return yield new Promise((resolve, reject) => {
58 |                 if (dataDriver === this.sendQuery) {
59 |                     this._pipe = net.createConnection('//./pipe/sp_remote_control', () => {
                                          ^
ENOENT: Failed to connect
   errno: 2
 syscall: "connect"

      at connect (node:net:369:10)
      at C:\Users\foorb\workspace\tsba\node_modules\soundpad.js\lib\esm\soundpad.js:59:38
      at new Promise (1:11)
      at C:\Users\foorb\workspace\tsba\node_modules\soundpad.js\lib\esm\soundpad.js:57:26
      at C:\Users\foorb\workspace\tsba\node_modules\soundpad.js\lib\esm\soundpad.js:7:71
      at new Promise (1:11)
      at __awaiter (C:\Users\foorb\workspace\tsba\node_modules\soundpad.js\lib\esm\soundpad.js:3:12)
      at C:\Users\foorb\workspace\tsba\test.ts:10:20
      at run (C:\Users\foorb\workspace\tsba\test.ts:7:19)

```

bun recently added support for pipes https://github.com/oven-sh/bun/pull/13838

My bun version: Bun v1.1.30-canary.111+50ed09654 (Windows x64)

I am not sure if this is a soundpad.js or bun problem.

With nodejs every worked.

**Comment by Seblor** [comment]
Hello !

Yeah, that's definitely a Bun issue. I noticed this behavior when I was working on the latest version. I don't think there's anything I can do on my end, and Bun's policy is that they should be 100% compatible with NodeJS out of the box, and anything that works on Node but doesn't on Bun is an issue on Bun, so you might have to submit a ticket to their repository.

> If you run into any bugs with a particular package, please [open an issue](https://bun.sh/issues). Opening issues for compatibility bugs helps us prioritize what to work on next.

https://bun.sh/docs/runtime/nodejs-apis

I'll try to find a workaround on my end the next couple days, but if I don't find anything, I'll close this issue by the end of the week.

**Comment by Seblor** [comment]
Quick update I found a fix and I will implement it, but IMO it seems to be an oversight on Bun's end.

The issue is simply that I used forward slashes for the named pipe path, and bun seems to only support backslashes...

**Comment by Seblor** [comment]
Fix has been implemented as of version 2.5.1. Please update your package version and it should be fixed.

Feel free to close this issue if it is properly working on your end.
